default.nix

```diff
@@ -38,13 +38,9 @@ rec {
 
   # TODO This stuff should probably be exposed as an overlay in the plutus-apps if
   # we switch to flakes.
-  webCommon = pkgs.callPackage (sources.plutus-apps + "/web-common") { inherit (marlowe.lib) gitignore-nix; };
+  webCommon = pkgs.callPackage sources.web-common { inherit (marlowe.lib) gitignore-nix; };
```

Comment:
where is the web-common coming from now? How can I override it with a local implementation?

Reply:
`web-common` is now its own repo - https://github.com/input-output-hk/purescript-web-common.

It exposes a standard package set that defines its self, from which we inherit in the project `packages.dhall`. You can override the PureScript dependencies locally like this:

```bash
# clone the web-common repo
$ git clone git@github.com:input-output-hk/purescript-web-common.git
$ cd purescript-web-common

# make local changes ...

# open the packages.dhall file in marlowe-cardano
$ cd ../marlowe-cardano
$ open ./packages.dhall
```

Override `web-common`:

```
let upstream =
      https://github.com/input-output-hk/purescript-web-common/releases/download/v1.1.4/packages.dhall sha256:ab905184ee034c35867e73ace0475ce29d1a0b975581d355af6f35e680449749

let overrides = upstream
  with web-common = ../purescript-web-common/spago.dhall as Location

let additions = {=}

in  upstream // overrides // additions
```

As for using a local version for the `static` files, for now you can update the entry in `shell.nix` that says `export WEB_COMMON_SRC="${webCommon}"` to `export WEB_COMMON_SRC=<absolute-path-to-web-common-repo>` And it should work with `npm run build:webpack:dev` /  `npm start`.

I am in the process of creating an `npm` module for `web-common`, after which you should be able to use `npm link` to manage this.